mods/ts/rules/defaults.yaml

```diff
@@ -382,6 +382,8 @@
 		ReferencePoint: Bottom, Right
 		RequiresCondition: hospitalheal
 	RevealOnFire:
+	RejectsOrders:
```

Comment:
I don't mind, it's just that I'd have to list every other order here, unless we add an explicit `Reject` field to this trait or an explicit `AcceptsOrders` counter-part trait.